test/cpp/naming/test_tcp_connect.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python2.7
```

Comment:
ditto wrt to not being really a test, more a util/aux bin.

Reply:
In the most recent commit I renamed 
`test_dns_server.py` to `dns_server.py`, 
`test_tcp_connect.py` to `tcp_connect.py`,
`test_dns_resolver.py` to `dns_resolver.py`

and moved all three to be under a `utils` subdirectory.

PTAL

Reply:
also added one line forgotten in #14609